encord/objects/ontology_labels_impl.py

```diff
@@ -2809,9 +2809,13 @@ def _set_answer_from_dict(self, answer_dict: Dict[str, Any], attribute: Attribut
         if isinstance(attribute, TextAttribute):
             self._set_answer_unsafe(answer_dict["answers"], attribute, track_hash, ranges)
         elif isinstance(attribute, RadioAttribute):
-            feature_hash = answer_dict["answers"][0]["featureHash"]
-            option = _get_option_by_hash(feature_hash, attribute.options)
-            self._set_answer_unsafe(option, attribute, track_hash, ranges)
+            if len(answer_dict["answers"]) == 1:
+                # When classification is removed in UI, it keeps the entry about the classification,
```

Comment:
Can this also happen for `Classification` objects? if yes, we'd need to do the same there I guess.

Also, keep in mind that now you're changing the label row if you download and upload, but this is probably fine.